### DISCUSSION TITLE
Github link for documents

### DISCUSSION THREAD

**Discussion by mirjak** [ietf-tools/datatracker#3925] [Ideas]
Currently we only have a field to add a link to GitHub on the group level, would be nice to be able to also add a GitHub link for each document.

**Comment by larseggert** [comment]
You have that already, by editing "Additional resources" for a document?

**Comment by mirjak** [comment]
Do we only have additional resources for individual draft or also for wg docs?

**Comment by larseggert** [comment]
We have that ability for all docs, I think.

**Comment by mirjak** [comment]
Okay, then I guess only the chairs can it for working groups documents. 

So what I was specifically looking for is a quick way to get to the GitHub repo from the document. In quic the GitHub link is provided in the wg about tap but not on each document page. I guess would be nice to add a link there as well (if no more specificities GitHub link is provided).
